scripts/generate_eval_report.py

```
@@ -0,0 +1,1255 @@
+                test_result["test_case"] = parts[0]
+                test_result["model"] = "-".join(parts[1:])
+            else:
+                test_result["test_case"] = params
+                test_result["model"] = "unknown"
+
+        results["tests"].append(test_result)
+
+    return results
+
+
+def generate_summary_table(results: Dict[str, Any], models: List[str]) -> str:
```

Ruff (0.12.2)

Unused function argument: `models`

(ARG001)